applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -216,14 +279,24 @@ u8_t hidc_read_cb(struct bt_gatt_hids_c *hidc,
 static bool event_handler(const struct event_header *eh)
 {
 	if (is_hid_report_sent_event(eh)) {
-		u32_t key = irq_lock();
-		if (next_event) {
-			EVENT_SUBMIT(next_event);
-			next_event = NULL;
+		k_spinlock_key_t key = k_spin_lock(&spinlock);
+		sys_snode_t *kbd_event_node = sys_slist_get(&keyboard_event_list);
+		struct keyboard_event_item *next_item = NULL;
```

Comment:
move `struct keyboard_event_item *next_item = NULL;` before the lock is taken to indicate that it is used outside the lock

Reply:
done